src/app/theme.ts

```diff
@@ -16,6 +16,21 @@ const textColors = {
   white: '#fff',
 };
 
+const contentBuffer = {
+  default: {
+    padding: '0 13.5rem',
+  },
+  toolbar: {
+    padding: '0 15.5rem',
```

Comment:
i didn't realize that the toolbar had a different width than everything else. lets just add the additional 2rem padding in the toolbar rather than having this here